src/server/rdb_save.cc

```diff
@@ -690,6 +691,19 @@ size_t RdbSerializer::SerializedLen() const {
   return mem_buf_.InputLen();
 }
 
+error_code RdbSerializer::WriteJournalEntries(absl::Span<const journal::Entry> entries) {
```

Comment:
I see only one place calling this method with 1 entry. Why not pass const& journal::Entry ?

Reply:
In the next PR the journal listener in dflycmd will be buffered, so it will be (according to current plans, but not sure) used there.

I can remove it, but it'll be lost efforts